functions/confirm_registration/main.py

```diff
@@ -1,19 +1,27 @@
+import json
 import logging
 import secrets
 from datetime import datetime, timedelta
 from typing import Optional
 
 import pytz
-from flask import jsonify
+from flask import jsonify, current_app
 from google.cloud import datastore
 from google.cloud.datastore import Entity
 
+current_app.config['JSON_AS_ASCII'] = False
 CONFIRMATIONS_PER_MSISDN_LIMIT = 3
 REGISTRATION_STATUS_COMPLETED = "completed"
 REGISTRATION_STATUS_INCORRECT = "incorrect"
 DATA_STORE_REGISTRATION_KIND = "Registrations"
 DATA_STORE_USERS_KIND = "Users"
 
+LANG = None
```

Comment:
Ustawienie globalnej zmiennej `LANG` nie wydaje mi się dobrym pomysłem. 

Patrząc po [późniejszym użyciu](https://github.com/ProteGO-app/backend/pull/56/files#diff-7ee2bffa260fc9306067ad9b7a3928e2R114): 

1. nazwa LANG sugeruje że jest to stała, co nie jest prawdą
2. ta zmienna może być [używana przez różne wywołania ten funkcji](https://cloud.google.com/functions/docs/bestpractices/tips#use_global_variables_to_reuse_objects_in_future_invocations). Mam obawy, że współbieżne wykonania mogą sobie podmieniać tą wartość 

(podobnie w innych funkcjach)